__tests__/app.test.js

```diff
@@ -8,193 +8,245 @@ beforeEach(() => seed(data));
 
-describe("GET ARTICLE BY ID", () => {
-  test("Responds with the article by ID", () => {
-    const article_id = 3;
-    return request(app)
-      .get(`/api/articles/${article_id}`)
-      .expect(200)
-      .then(({ body }) => {
-        expect(body.article).toEqual({
-          article_id: 3,
+//////GET ARTICLES//////
 
-          title: "Eight pug gifs that remind me of mitch",
-          topic: "mitch",
-          author: "icellusedkars",
-          body: "some gifs",
-          created_at: "2020-11-03T09:12:00.000Z",
-          votes: 0,
-          comment_count: "2",
+describe("GET /api/articles", () => {
+  describe("Happy Paths", () => {
+    test("Responds with an object containing a key of articles with array of articles", () => {
+      return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toHaveLength(12);
+          console.log(body.articles, "check this");
+          body.articles.forEach((article) => {
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(Number),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_At: expect.any(Number),
+              votes: expect.any(Number),
+            });
+          });
         });
+    });
+    describe("Error Handling", () => {
+      test("404 message: Returns 404 error if path is invalid", () => {
+        return request(app)
+          .get("/api/notauser")
+          .expect(404)
+          .then(({ body: { message } }) => {
+            expect(message).toBe("Path not found");
+          });
       });
+    });
   });
-  describe("GET ARTICLE BY ID ERROR HANDLING", () => {
-    test("404 message: Returns 404 error if article doesn't exist", () => {
-      const article_id = 13;
+});
+
+describe("GET /api/articles/:article_id", () => {
+  describe("Happy Paths", () => {
+    test("Responds with the article by ID", () => {
+      const article_id = 3;
       return request(app)
         .get(`/api/articles/${article_id}`)
-        .expect(404)
-        .then(({ body: { message } }) => {
-          expect(message).toEqual("Article not found!");
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toEqual({
+            article_id: 3,
+            title: "Eight pug gifs that remind me of mitch",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "some gifs",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 0,
+            comment_count: "2",
+          });
         });
     });
-    test("400 message: Returns 400 error if data type is incorrect", () => {
-      const article_id = "abc";
+    test("Responds with article including comment count", () => {
+      const article_id = 5;
       return request(app)
         .get(`/api/articles/${article_id}`)
```

Comment:
You've already testing this in the previous test. Consider testing an article that has 0 comments instead to make sure your JOIN works ok